Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM --platform=$BUILDPLATFORM golang:1.19-alpine as builder
 ARG VERSION
-RUN apk --no-cache add ca-certificates
+RUN adduser -u 1001 -D valkyrie && apk --no-cache add ca-certificates=20220614-r4
```

Comment:
Also tried upgrading version here (since there seem to be newer ca-certificates versions), but apk wouldn't let me, seems like something else depends on lower version. Anyway, this is the version that comes with base image, and the same version is kept after running apk.